src/amo/routes.js

```diff
@@ -42,7 +43,13 @@ export default (
       <IndexRoute component={Home} />
       <Route path="addon/:slug/" component={Addon} />
       <Route path="addon/:addonSlug/reviews/" component={AddonReviewList} />
-      <Route path="user/:username/" component={UserProfile} />
+      {config.get('enableUserProfile') && (
+        <React.Fragment>
+          <Route path="users/edit" component={UserProfileEdit} />
+          <Route path="user/:username/edit" component={UserProfileEdit} />
+          <Route path="user/:username/" component={UserProfile} />
```

Comment:
This was not previously wrapped in an `if` based on `enableUserProfile`. Should we keep it outside of the `if`?

Reply:
Yes, although we do not have a nginx rule for that in -prod or -stage so it is almost the same result.

Reply:
I actually believe this was a mistake, isnt it? I believe the flag should control the whole user profile feature.

Reply:
I thought we already had the new User Profile view available on prod, but no, we don't, so I think you're correct that the original was a mistake and it all can be wrapped in the `if`.